pkg/server/server.go

```diff
@@ -330,11 +330,15 @@ func (s *Server) newEndpointsServer(ctx context.Context, catalog catalog.Catalog
 }
 
 func (s *Server) newBundleManager(cat catalog.Catalog, metrics telemetry.Metrics) *bundle_client.Manager {
+	log := s.config.Log.WithField(telemetry.SubsystemName, "bundle_client")
 	return bundle_client.NewManager(bundle_client.ManagerConfig{
-		Log:       s.config.Log.WithField(telemetry.SubsystemName, "bundle_client"),
+		Log:       log,
 		Metrics:   metrics,
 		DataStore: cat.GetDataStore(),
-		Source:    bundle_client.TrustDomainConfigMap(s.config.Federation.FederatesWith),
+		Source: bundle_client.MergeTrustDomainConfigSources(
+			bundle_client.TrustDomainConfigMap(s.config.Federation.FederatesWith),
+			bundle_client.DataStoreTrustDomainConfigSource(log, cat.GetDataStore()),
```

Comment:
I don't recall what we decided here in terms of which source would get priority. As written, the static configuration will overwrite datastore results.

Reply:
That's what we decided!